tests/test_wfs3_pygeoapi.py

```diff
@@ -34,7 +34,11 @@ def test_wfs3_pygeoapi():
     assert lakes['title'] == 'Large Lakes'
     assert lakes['description'] == 'lakes of the world, public domain'
 
+    # Minimum of limit param is 1
```

Comment:
Note these fixes are already in master via 9f7bf60746d9bcc1c7a3646f80f6478b89305c8d

Reply:
ok, will integrate.